src/kvstore.c

```diff
@@ -191,28 +181,10 @@ static void kvstoreDictBucketChanged(dict *d, long long delta) {
     kvs->bucket_count += delta;
 }
 
-/* Returns the size of the DB dict base metadata in bytes. */
-static size_t kvstoreDictMetaBaseSize(dict *d) {
-    UNUSED(d);
-    return sizeof(kvstoreDictMetaBase);
-}
 /* Returns the size of the DB dict extended metadata in bytes. */
-static size_t kvstoreDictMetadataExtendSize(dict *d) {
+static size_t kvstoreDictBaseMetadataSize(dict *d) {
```

Comment:
```suggestion
static size_t kvstoreDictBaseMetaSize(dict *d) {
```
IMHO the word meta is rather clear.

Reply:
Done.